pulsar-broker/src/test/java/org/apache/pulsar/client/api/InterceptorsTest.java

```diff
@@ -856,6 +855,91 @@ public void onPartitionsChange(String topicName, int partitions) {
         Assert.assertNull(reader.readNext(3, TimeUnit.SECONDS));
     }
 
+    @Test
+    public void testConsumerInterceptorWithMultiTopics() throws PulsarClientException {
```

Comment:
We already have the test `testConsumerInterceptorWithMultiTopicSubscribe` but it looks like the test can't cover this case. It's better to update the existing one to cover this case without maintaining to similar tests.

Reply:
Ok, improved test.